joss/paper.md

```diff
@@ -0,0 +1,131 @@
+  index: 3
+- name: SETI Institute, Mountain View, CA 94043, USA
+  index: 4
+- name: Lockheed Martin Solar and Astrophysics Laboratory, Palo Alto, CA 94304, USA
+  index: 5
+- name: Air Force Research Laboratory, Space Vehicles Directorate, Kirtland AFB, NM 87117, USA
+  index: 6
+- name: Space and Atmospheric Physics, The Blackett Laboratory, Imperial College, London SW7 2BW, UK
+  index: 7
+- name: School of Mathematics and Statistics, University of St Andrews, St Andrews KY16 9SS, UK
+  index: 8
+- name: Department of Physics and Astronomy, Rice University, Houston, TX 77005, USA
+  index: 9
+date: 18 October 2025
+bibliography: paper.bib
+---
+
+# Summary
+
+Understanding the response of the plasma in the solar corona, the outermost layer of the Sun's atmosphere, to heating is of considerable importance in understanding flares and the heating of the quiescent corona.
+This requires detailed numerical modeling of the coupled system of the coronal plasma and the solar magnetic field[^stellar].
+While solving the full set of three-dimensional *magnetohydrodynamic* (MHD) equations is feasible for small regions of the corona, the needed computational resources and physical complexity of such models means they are not always amenable to simple or even correct interpretation.
+Field-aligned hydrodynamic models [e.g. HYDRAD @bradshaw_self-consistent_2003] exploit the fact that the magnetic pressure in the corona is much greater than the gas pressure such that the corona can be considered as a series of mini atmospheres, or *coronal loops*, where the plasma responds hydrodynamically to the heating and traces out the complex coronal magnetic field.
+As such, the explicit dependence on the magnetic field can be neglected and the relevant hydrodynamic equations can be reduced to a single-dimension in space: the coordinate along the coronal loop.
+However, the large range of spatial and temporal scales necessary to resolve this system, in particular the severe time step limitations imposed by thermal conduction, mean that even field-aligned models are computationally expensive enough to make large parameter explorations prohibitive.
+The enthalpy-based thermal evolution of loops (EBTEL) model [@klimchuk_highly_2008;@cargill_enthalpy-based_2012] was originally developed in order to provide a simple and efficient way to study the coronal plasma response to time-dependent plasma heating.
+EBTEL accomplishes this by computing spatial integrals of the aforementioned field-aligned hydrodynamic equations.
+Comparisons to spatially-averaged results from field-aligned hydrodynamic models show very good agreement [@cargill_enthalpy-based_2012].
+Because of its relative simplicity and computational efficiency, EBTEL has been widely used since its initial development [e.g. @qiu_heating_2012;@ugarte-urra_determining_2014] with @klimchuk_highly_2008 and @cargill_enthalpy-based_2012 having nearly 400 citations combined.
```

Comment:
Maybe change `nearly 400 citations` to `hundreds of citations combined` or something along those lines?  The number will grow with time!